src/components/Filter/components/FilterSectionHeader/filterSectionHeaderUtil.tsx

```diff
@@ -17,11 +17,13 @@ export const createChips = (value: string | string[]) => {
                     return null;
                 }
 
-                return (
-                    <FilterValueChip key={`${content}-${idx}`}>
-                        <div>{content}</div>
-                    </FilterValueChip>
-                );
+                return <FilterValueChip key={`${content}-${idx}`}>
+                    <div>
+                        {content}
+                    </div>
+                    <FilterLabels>{label}</FilterLabels>
+                    {showX && onClose && <span onClick={() => onClose(name, content)}>x</span>}
```

Comment:
I think we may need to use an actual "X" asset/icon/svg instead of text here. The text "x" proportions are uneven. You could use this one: https://github.com/ToyotaResearchInstitute/lakefront/blob/main/src/components/Modal/assets/closeIcon.svg